Part1_InterruptControllersEvolution.html

```diff
@@ -3,25 +3,25 @@
 <html lang="en">
 <head>
 	<meta charset="utf-8">
-	<title>External Interrupts in x86 system. Part 1. Interrupt controllers evolution</title>
+	<title>External Interrupts in the x86 system. Part 1. Interrupt controller evolution</title>
 </head>
 
 <body>
-	<h1>External Interrupts in x86 system. Part 1. Interrupt controller evolution</h1>
+	<h1>External Interrupts in the x86 system. Part 1. Interrupt controller evolution</h1>
 
-	<p>This article is about an interrupt delivery process from external devices in x86 system and tries to answer questions such as:<p/>
+	<p>This article is about the interrupt delivery process from external devices in the x86 system. It tries to answer questions such as:<p/>
 
 	<ul>
-		<li>what is PIC and what is it for?</li>
-		<li>what is APIC and what is it for? What is a purpose of LAPIC and I/O APIC?</li>
-		<li>what are the differences between APIC, xAPIC and x2APIC?</li>
-		<li>what is MSI? What are the differences between MSI and MSI-X?</li>
-		<li>what is a role of tables $PIR, MPtable, ACPI?</li>
+		<li>What is PIC and what is it for?</li>
+		<li>What is APIC and what is it for? What is the purpose of LAPIC and I/O APIC?</li>
+		<li>What are the differences between APIC, xAPIC, and x2APIC?</li>

```

Comment:
Hmm, I started to look for an explanation to your extra comma and have found this article https://en.wikipedia.org/wiki/Serial_comma I started to look, cause it is not necessary in my language. It seems like it is optional in English

Reply:
It is optional, but I added it in for consistency with the sentence two lines below. I also personally like it :)